apps/test/unit/code-studio/pd/workshop_dashboard/EnrollmentsPanelTest.jsx

```diff
@@ -52,7 +52,7 @@ describe('EnrollmentsPanel', () => {
     assert(!wrapper.find('Spinner').exists(), 'Spinner was not rendered');
     assert(
       wrapper.find('WorkshopEnrollment').exists(),
-      'WorkshopEnrollment was rendered'
+      'WorkshopEnrollment was not rendered'
```

Comment:
I think this is supposed to tell people what went wrong?

Reply:
The [chai library](https://www.chaijs.com/api/assert/) isn't opinionated here; it just describes the argument as "a custom error message to show when the assertion fails."  I've seen teams do these both ways; we aren't consistent.  As long as the output is useful to you when the test fails, this seems fine (and consistency at least within the file seems good).